### PR TITLE
Feature/leminh wish list

### PR DESCRIPTION
**Pull request by pvlminh3005**
Create 3 pages: 
+ Checkout
+ My Order
+ Whishlist

### CODE REVIEW

File: lib/presentation/pages/checkout/checkout_page.dart [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+import 'package:furniture_shop/values/font_sizes.dart';
+
+class CheckOutPage extends StatelessWidget {
+  const CheckOutPage({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: _buildAppBar(),
+      body: _buildBody(),
```
Comment (hahpuc) [inline lib/presentation/pages/checkout/checkout_page.dart:21]: Add to scroll screen becuz if using small screen, the content cannot be seen

File: lib/presentation/pages/checkout/checkout_page.dart [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+  }
+
+  Widget _buildDetailList() {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        _buildTitle('Shipping Address'),
+        _buildDetailShippingAddress(),
+        _buildTitle('Payment'),
+        __buildDetailPayment(),
```
Comment (hahpuc) [inline lib/presentation/pages/checkout/checkout_page.dart:63]: I think should use button to change payment method

File: lib/presentation/pages/checkout/widgets/info_price_widget.dart [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  }
+
+  Widget _buildPrice() {
+    return RichText(
+      text: TextSpan(
+        text: discount!
+            ? '-${price!.toStringAsFixed(2)}'
+            : '${price!.toStringAsFixed(2)}',
+        style: TextStyle(
+          fontSize: FontSize.BIG,
```
Comment (hahpuc) [inline lib/presentation/pages/checkout/widgets/info_price_widget.dart:50]: Add FontFamily: Nunito Sans

File: lib/presentation/pages/order/widgets/order_cart.dart [hunk 4]
```diff
@@ -0,0 +1,164 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:furniture_shop/presentation/widgets/base/custom_text.dart';
+import 'package:furniture_shop/values/colors.dart';
+import 'package:furniture_shop/values/dimens.dart';
+import 'package:furniture_shop/values/font_sizes.dart';
+
+class OrderCard extends StatelessWidget {
```
Comment (hahpuc) [inline lib/presentation/pages/order/widgets/order_cart.dart:8]: Add Button into OrderCard (to navigate to order detail)

File: lib/presentation/pages/whish_list/whish_list_page.dart [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: _buildAppBar(),
+      body: _buildBody(),
+    );
+  }
+
+  CustomAppBar _buildAppBar() {
+    return CustomAppBar(
+      title: Text('My cart'),
```
Comment (hahpuc) [inline lib/presentation/pages/whish_list/whish_list_page.dart]: Change FontFamily: Gelasio

File: lib/presentation/pages/whish_list/widget/whish_list_widget.dart [hunk 6]
```diff
@@ -0,0 +1,233 @@
...
+        shape: BoxShape.circle,
+        color: statusWhish == 1 ? Colors.white : Colors.black,
+        border: Border.all(
+          width: 1,
+          color: statusWhish == 1 ? AppColor.colorGrey : Colors.transparent,
+        ),
+      ),
+      child: Padding(
+        padding: const EdgeInsets.all(3.0),
+        child: Icon(
```
Comment (hahpuc) [inline lib/presentation/pages/whish_list/widget/whish_list_widget.dart]: Change to IconButton (Tap to choose product)

**Review by hahpuc — COMMENTED** [review]
Your code is OK

File: lib/presentation/pages/checkout/checkout_page.dart (hunk 1 above)
Comment (pvlminh3005) [reply lib/presentation/pages/checkout/checkout_page.dart:21]: done

File: lib/presentation/pages/checkout/checkout_page.dart (hunk 2 above)
Comment (pvlminh3005) [reply lib/presentation/pages/checkout/checkout_page.dart:63]: done

File: lib/presentation/pages/checkout/widgets/info_price_widget.dart (hunk 3 above)
Comment (pvlminh3005) [reply lib/presentation/pages/checkout/widgets/info_price_widget.dart:50]: done

File: lib/presentation/pages/order/widgets/order_cart.dart (hunk 4 above)
Comment (pvlminh3005) [reply lib/presentation/pages/order/widgets/order_cart.dart:8]: done

File: lib/presentation/pages/whish_list/whish_list_page.dart (hunk 5 above)
Comment (pvlminh3005) [reply lib/presentation/pages/whish_list/whish_list_page.dart]: done

File: lib/presentation/pages/whish_list/widget/whish_list_widget.dart (hunk 6 above)
Comment (pvlminh3005) [reply lib/presentation/pages/whish_list/widget/whish_list_widget.dart]: done

**Review by hahpuc — APPROVED** [review]

**Pull request merged**